packages/presentation/lib/l10n/intl_ru.arb

```diff
@@ -0,0 +1,3 @@
+{
+  "share": "Смотри, какой я фильм нашел! \n\nhttps://www.themoviedb.org/movie/{movie_id}?language={language}"
```

Comment:
why u have only one line here? In en version u have 16 variables

Reply:
according to the assignment, we don’t have such a thing as adding a Russian locale to the entire application